FrontEnd/composables/useUser.js

```diff
@@ -1,76 +1,75 @@
 // composables/useAuth.js
-// import { useAuthStore } from '@/store/auth';
-import { useUserStore } from "~/stores/useUserStore";
-import { useNuxtApp } from "#app";
-import { srvLogin, srvSignup } from "~/services/authServices";
-import { srvChangePropic, srvGetWhoToFollow, srvHandleFollow, srvHandleProfile } from "~/services/userServices";
+import { userStore } from "~/stores/userStore";
+import {srvGetWhoToFollow,srvHandleFollow,srvHandleProfile,} from "~/services/api/userServices"
+
 
 export const useUser = () => {
-  const { $api } = useNuxtApp();
-  //   const authStore = useAuthStore();
-  const userStore = useUserStore();
+  const { $customApi } = useNuxtApp();
+  const { $toast } = useNuxtApp();
+
 
-//   const handleAuthError = (error) => {
-//     console.error("Authentication error: ", error.message);
-//   };
 
   const whoTofollow = async () => {
     try {
-        const response = await srvGetWhoToFollow($api);
```

Comment:
srvGetWhoToFollow - this is okay but not good naming for the function. Also, functions need not have srv in the beginning. If its getting across network, a better name would be _fetchFollowSuggestions_

Reply:
I have updated the services functions with proper naming conventions,please review the latest commit for changes.